Capgemini.Xrm.CdsDataMigrator/Capgemini.Xrm.CdsDataMigratorLibrary.Tests.Unit/Presenters/ExportPresenterTests.cs

```diff
@@ -108,13 +109,38 @@ public void ExportDataActionDoesNotThrowException()
             dataMigrationService.Setup(a => a.ExportData(It.IsAny<ExportSettings>()));
             logger.Setup(a => a.LogError(It.IsAny<string>()));
 
-            FluentActions.Invoking(() => systemUnderTest.ExportDataAction())
+            FluentActions.Invoking(() => systemUnderTest./*ExportDataAction*/ExportData(null, new EventArgs()))
```

Comment:
Can this comment be removed? 
```suggestion
            FluentActions.Invoking(() => systemUnderTest.ExportData(null, new EventArgs()))
```

Reply:
addressed in latest commit